docs/contributing/community.md

```diff
@@ -27,6 +27,11 @@ Everyone is welcome to join & add items to the agenda. The meeting happens in Zo
 
 If you have any suggestions, how we can run the meeting better, please let `@povilasv` or `@metalmatze` know in the CNCF slack.
 
+### Thanos Contributor Office Hours
+
+Thanos Contributor Office Hours are public weekly meetings focused on Thanos development and contributions. It’s meant for developers and maintainers to meet and get unblocked, pair review, and discuss development aspects of the Thanos and related projects. Read more [here](https://github.com/thanos-io/thanos/tree/master/docs/contributing/Thanos-Contributor-Office-Hours.md)
```

Comment:
```suggestion
Thanos Contributor Office Hours are public weekly meetings focused on Thanos development and contributions. It’s meant for developers and maintainers to meet and get unblocked, pair review, and discuss development aspects of the Thanos and related projects. Read more [here](https://docs.google.com/document/d/137XnxfOT2p1NcNUq6NWZjwmtlSdA6Wyti86Pd6cyQhs/edit#heading=h.oy8b2jtupl8m)
```
Can we link Google doc instead?

Reply:
Yes, we can I have done the requested changes in PR #3552